[Term Entry] UI and UX Design UIUX (Ideation) Crazy Eights #4412 (#4442)
Codecademy/docs: content/uiux/concepts/crazy-eights/crazy-eights.md

---
Title: 'Crazy Eights'

> Description: 'A sketching exercise that challenges people to sketch eight distinct ideas in eight minutes'

Reviewer: ```suggestion
Description: 'A quick sketching exercise that challenges people to sketch eight distinct ideas in eight minutes.'
```

Author: done

> CatalogContent:

Reviewer: Refer [this uiux entry](https://github.com/Codecademy/docs/blob/main/content/uiux/concepts/design-methodologies/terms/five-elements-of-ux-design/five-elements-of-ux-design.md?plain=1) and add `Subjects` and  `Tags` to this entries metadata.

Author: done

  - 'intro-to-ui-ux'
  - 'paths/front-end-engineer-career-path'
---

> **Crazy Eights's** a Design Sprint method: meaning it is a quick sketching exercise that challenges people to sketch eight different ideas in eight minutes. The goal is to push beyond your first idea, which is usually basic(the least innovative), and to generate a wide variety of solutions to your initial design.

Reviewer: ```suggestion
**Crazy Eights** is a Design Sprint method used as a quick sketching exercise that challenges people to sketch eight distinct ideas in eight minutes, for the problem at hand. The goal is to push beyond the preliminary idea, and to generate a wide variety of solutions.
```
We refrain from using any personal pronouns like you, we, .. in the entry.

Author: done

> **Crazy Eights's** is a core Design Sprint method. It is a fast sketching exercise that challenges people to sketch eight distinct ideas in eight minutes. The goal is to push beyond your first idea, which is frequently the basic(least innovative), and to generate a wide variety of solutions to your design(challenge).

Reviewer: This seems to be taken directly from here:
https://designsprintkit.withgoogle.com/methodology/phase3-sketch/crazy-8s

Author: nah I didn't get that information from, seems like many sources have the same verbiage on the topic. No worries ill take core of it.

> #### Example  

Reviewer: ```suggestion
## Steps in Crazy Eights 
```

Author: done

> To properly implement the crazy eight method would be to 1st begin with an idea in mind for your design then setting a 8minute timer and folding a piece of paper 3 times so that you have eight boxes you then start the timer begin sketching and spend no more than 1 minute per box. Do this with other members in your team the more the ideas the better since this is just the drafting stage.

Reviewer: ```suggestion
The Crazy Eights methodology is used in a team setting to gather a multitude of ideas. To implement this method follow the steps below: 
1. Begin with a problem statement to solve for.
2. Take a sheet of paper and divide it into 8 sections.
3. Set a 8 minute timer.
4. Start the timer and begin sketching solutions for the problem at hand. 

> **Note:** Spend no more than 1 minute per idea. 

Once the activity is complete, the ideas generated by the entire team are used to finalize a solution.
```
I have structured this example to be steps for crazy eights as it makes more sense for this entry and removed other grammatical errors.

Author: done.

> ![crazy 8's example #1](<www.github.com/codecademy/docs/media/crazy8's #1.jpg>) ![8minTimer](www.github.com/codecademy/docs/media/8mintimer.jpg) ![crazy 8's example #2](<www.github.com/codecademy/docs/media/crazy8's #2.png>)

Reviewer: I think only one of the crazy 8's example image should be enough. Please remove the other image and the `8mintimer.jpg` image. Also remember to remove these images from the media folder.

Author: done

---

> Title: 'Crazy Eights in  UI/UX'

Reviewer: ```suggestion
Title: 'Crazy Eights'
```

Author: done

> Description: 'It is a fast sketching exercise that challenges people to sketch eight distinct ideas in eight minutes'

Reviewer: ```suggestion
Description: 'A sketching exercise that challenges people to sketch eight distinct ideas in eight minutes.'
```

Author: done

CatalogContent:
  - 'intro-to-ui-ux'
  - 'paths/front-end-engineer-career-path'
---

> **Crazy Eights In UX**

Reviewer: ```suggestion
```
Not necessary to separately add as a title

Author: done

> Crazy 8's is a core Design Sprint method. It is a fast sketching exercise that challenges people to sketch eight distinct ideas in eight minutes. The goal is to push beyond your first idea, which is frequently the basic(least innovative), and to generate a wide variety of solutions to your design(challenge).

Reviewer: ```suggestion
**Crazy Eight's** is a core Design Sprint method. It is a fast sketching exercise that challenges people to sketch eight distinct ideas in eight minutes. The goal is to push beyond your first idea, which is frequently the basic(least innovative), and to generate a wide variety of solutions to your design(challenge).
```

Author: done
